skimage/draw/draw.py

```diff
@@ -712,8 +712,8 @@ def rectangle(origin, extent, shape=None):
 
     Returns
     -------
-    coords : (M, n) ndarray of int with the
-        Pixel coordinates of the nd rectangle.
+    coords : (M, N)
```

Comment:
This still needs a bit of tweaking (see other docstrings in the library). eg:

```
coords: array of int, shape (Ndim, Npoints)
    The coordinates of all pixels in the nD rectangle.
```

Leave the "e.g." for the Examples section.